package.json

```diff
@@ -19,6 +19,7 @@
     "web-vitals": "^1.0.1"
   },
   "devDependencies": {
+    "@babel/core": "^7.14.3",
```

Comment:
ini ad tambahan buat package dipake untuk ap?

Reply:
apakah harus dihapus ?

Reply:
@ariefrahmansyah do we need to add this package?

Reply:
Gapapa tambahin aja. Ini devDependencies juga soalnya